docs/apis/api-blockchain-get-current-mini-epoch.md

```diff
@@ -0,0 +1,87 @@
+---
+id: api-blockchain-get-current-mini-epoch
+title: GetCurrentMiniEpoch
+---
+
+---
+
+Returns the current TX block number of the network. This is represented as a `String`.
+
+### Example Request
+
+=== "cURL"
+
+    ```shell
```

Comment:
So much duplication of information here. I'm looking forward to trying to auto-generate lots of this from a spec :)

Reply:
Yes indeed. There is still _a lot_ of clean up and automation to do.